hollow/src/main/java/com/netflix/hollow/api/consumer/HollowConsumer.java

```diff
@@ -163,6 +163,25 @@ protected HollowConsumer(BlobRetriever blobRetriever,
             announcementWatcher.subscribeToUpdates(this);
     }
 
+    //// INCUBATING ////
+
+    protected HollowConsumer(AnnouncementWatcher announcementWatcher,
+                             HollowClientUpdater updater,
+                             Executor refreshExecutor,
+                             HollowConsumerMetrics metrics,
+                             ReadWriteLock refreshLock) {
+        /*
+         * notably different, this constructor doesn't auto-subscribe to announcement watcher
```

Comment:
We've received some feedback that the subscribe to automatic background updates on consumer construction violated the principle of least astonishment.

We also have a Cinder use-case where we want more control over when and how the initial load happens even when using an `AnnouncementWatcher`.

Reply:
> We've received some feedback that the subscribe to automatic background updates on consumer construction violated the principle of least astonishment.

What would be the point of adding an `AnnouncementWatcher` if not to subscribe to automatic background updates?